### PR TITLE
For own code review

### PR DESCRIPTION
**Pull request by cosywasswa**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+body{
```
Comment (cosywasswa) [inline css/style.css:1]: Kindly use linters to detect errors in your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script  src="js/index.js"></script>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (cosywasswa) [inline index.html:6]: Kindly customize your project tittle to show what your project is all about.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  <body>
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
+      <h3>Former President Of South Africa</h3>
+
+      <div id="img-div">
+        <img id = "image" src="images/nelson mandela.jpg" alt="a picture of Nelson Mandela">
+        <p id="img-caption">A picture of one of Africa's revoluttionaries. The late papa Madiba.</p>
+      </div>
+
```
Comment (cosywasswa) [inline index.html:20]: Please use semantic tags eg (Header) in your project, this will make it look professional.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,5 @@
-# Code-review-repo
-To do a own code review for an admission trial project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (cosywasswa) [inline README.md:5]: Kindly customize this README file to describe what your project is all about.

**Review by cosywasswa — COMMENTED** [review]
Good job sofar , You are a one step to finish your project. there are a few changes needed for you to look at and correct them.
Thanks.

File: README.md (hunk 4 above)
Comment (wayungi) [reply README.md:5]: - [ ] Good comment. It is polite and clear 👏🏾

File: css/style.css (hunk 1 above)
Comment (wayungi) [reply css/style.css:1]: - [ ] Polite comment that clearly states why a change is required 👏🏾

File: index.html (hunk 2 above)
Comment (wayungi) [reply index.html:6]: - [ ] Polite, clear comment explaining why change is required 🥳

File: index.html (hunk 3 above)
Comment (wayungi) [reply index.html:20]: - [ ] Polite comment ✔️. Semantic tags also give context to screen readers and enable a vast audience of people with disabilities to use your website easily

**Review by wayungi — APPROVED** [review]
Hi @cosywasswa , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Good use of the "Review changes" feature from GitHub ✔️ 
- [ ] Wrote 2 or more comments using inline feature ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**